PMSCore.Test/AdminClassTest.cs

```diff
@@ -0,0 +1,166 @@
+﻿namespace PMSCore.Test
+
+{
+    public class AdminTests
+    {
+        [Fact]
+        public void CreateUser_ShouldCreateATester_WhenTesterPermissionIsApplied()
+        {
+            // Arrange - creating an Admin instance
+            var admin = new Admin("adminUser");
+
+            // Act - calling the CreateUser method
+            var user = admin.CreateUser("testerUser", Permission.TESTER);
+
```

Comment:
for this line: var user = admin.CreateUser("testerUser", Permission.TESTER); 
you can add a test that would check for duplicate user prevention